features/todo.feature

```diff
@@ -6,3 +6,8 @@ Feature: Todo API
     Given a todo exists
     When I retrieve the list of todos
     Then I will get a response with a list containing a todo
+
+  Scenario: Get a todo
```

Comment:
We should run the integrations specs via CI.

Reply:
4d40c5d3473a67df8c118e7ec46d95650ba2bd38